services/src/main/java/org/fao/geonet/api/records/MetadataInsertDeleteApi.java

```diff
@@ -474,8 +479,7 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
         dataManager.activateWorkflowIfConfigured(context, newId, group);
 
         try {
-            copyDataDir(context, sourceMetadata.getId(), newId, Params.Access.PUBLIC);
-            copyDataDir(context, sourceMetadata.getId(), newId, Params.Access.PRIVATE);
+            StoreUtils.copyDataDir(context, sourceMetadata.getId(), Integer.parseInt(newId), true);
```

Comment:
Creating a record with an Editor using a template that the user can not download fails with 
```{"message":"SecurityException","code":"forbidden","description":"Current user can't download resources for metadata 'e24b89cc-2e66-49e7-84b6-19f1ab106128' and as such can't access the requested resource."}```

StoreUtils.copyDataDir may return SecurityException now and creation fails.